Update creating-a-github-pages-site-with-jekyll.md (#2178)
github/docs: content/github/working-with-github-pages/creating-a-github-pages-site-with-jekyll.md

 $ git checkout --orphan gh-pages
 # Creates a new branch, with no history or contents, called gh-pages and switches to the gh-pages branch
 ```
7. To create a new Jekyll site, use the `jekyll new` command:
   ```shell
   $ jekyll new .
   # Creates a Jekyll site in the current directory
   ```

> 8. Open the Gemfile that was created by jekyll.

Reviewer: ```suggestion
8. Open the Gemfile that Jekyll created.
```

> 9. Add the github-pages gem: update the line starting with `# gem "github-pages"`so that it looks like this, replacing _GITHUB-PAGES-VERSION_ with the latest supported version of the `github-pages` gem. This can be found on the "[Dependency versions](https://pages.github.com/versions/)" page of the {% data variables.product.prodname_pages %} site:
>    ```shell
>    gem "github-pages", "~> GITHUB-PAGES-VERSION", group: :jekyll_plugins
>    ```
>    Comment out the Gemfile line which starts with:
>   ```shell
>    gem "jekyll"
>    ```

Reviewer: As the `gem jekyll ...` line comes first in the file, I think it would be good to swap round the 2 edits you have to do in this file:

```suggestion
1. Add "#" to the beginning of the line that starts with `gem "jekyll"` to comment out this line.
1. Add the `github-pages` gem by editing the line starting with `# gem "github-pages"`. Change this line to:

   ```shell
   gem "github-pages", "~> GITHUB-PAGES-VERSION", group: :jekyll_plugins```

Replace _GITHUB-PAGES-VERSION_ with the latest supported version of the `github-pages` gem. You can find this version here: "[Dependency versions](https://pages.github.com/versions/)."

```

 $ git checkout --orphan gh-pages
 # Creates a new branch, with no history or contents, called gh-pages and switches to the gh-pages branch
 ```
7. To create a new Jekyll site, use the `jekyll new` command:
   ```shell
   $ jekyll new .
   # Creates a Jekyll site in the current directory
   ```
8. Open the file called `Gemfile` that was created, and find the line that looks like this:
   ```shell
   # gem "github-pages", group: :jekyll_plugins
   ```
   and update it so that it looks like this:
   ```shell
   gem "github-pages", "~> 209", group: :jekyll_plugins
   ```

>    If there is a newer version of the `github-pages` gem, replace `209` above with the latest version that is listed in [Rubygems](https://rubygems.org/gems/github-pages).

Reviewer 2: I don't think it is good practice to include explicit version numbers which likely will not match what the user sees.

Instructing the user to `find` a specific line is a bit verbose without adding much information.

I think the target audience for this guide should already know that "the file called Gemfile" is typically just known as a "Gemfile". However, the target audience is not expected to know much about jekyll so I think it may be useful to specify that jekyll creates the Gemfile.

Retaining the previous wording but specifying that the version number should be the version of the github-pages gem and where that information can be found is a better solution IMO. To disambiguate with the previously used `VERSION` identifier and help avoid the confusion identified in #2177, we can use the clearer if more verbose `GITHUB-PAGES-VERSION`.
~~~suggestion
8. Open the Gemfile that was created by jekyll.
9. Add the github-pages gem: update the line starting with `# gem "github-pages"`so that it looks like this, replacing _GITHUB-PAGES-VERSION_ with the latest supported version of the `github-pages` gem. This can be found on the "[Dependency versions](https://pages.github.com/versions/)" page of the {% data variables.product.prodname_pages %} site:
   ```shell
   gem "github-pages", "~> GITHUB-PAGES-VERSION", group: :jekyll_plugins
   ```
~~~

   Then find the line that looks like this:
   ```shell
   gem "jekyll", "~> 4.2.0"
   ```
   and disable it by adding a `#` to it:
   ```shell
   # gem "jekyll", "~> 4.2.0"

>    ```

Reviewer 2: I don't think it is good practice to include explicit version numbers which likely will not match what the user sees.
~~~suggestion
   Comment out the Gemfile line which starts with:
  ```shell
   gem "jekyll"
   ```
  The correct version Jekyll will be installed as a dependency of the `github-pages` gem.
~~~
I think instructing users on how to comment out lines in a Gemfile is out of scope for this guide so this should be sufficient.

 $ git checkout --orphan gh-pages
 # Creates a new branch, with no history or contents, called gh-pages and switches to the gh-pages branch
 ```
7. To create a new Jekyll site, use the `jekyll new` command:
   ```shell
   $ jekyll new .
   # Creates a Jekyll site in the current directory
   ```
8. Open the Gemfile that Jekyll created.
1. Add "#" to the beginning of the line that starts with `gem "jekyll"` to comment out this line.
1. Add the `github-pages` gem by editing the line starting with `# gem "github-pages"`. Change this line to:

   ```shell

>    gem "github-pages", "~> GITHUB-PAGES-VERSION", group: :jekyll_plugins```

Reviewer: ```suggestion
   gem "github-pages", "~> GITHUB-PAGES-VERSION", group: :jekyll_plugins
   ```
```
